src/Eloquent/Concerns/Authorizable.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Bakery\Eloquent;
+
+use Illuminate\Support\Facades\Gate;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Auth\Access\AuthorizationException;
+
+trait Authorizable
+{
+    /**
+     * Determine if the current user can create a new model of throw an exception.
```

Comment:
```suggestion
     * Determine if the current user can create a new model or throw an exception.
```

Reply:
Also, the other methods use the general `authorize` and `authorized` method. Shouldn't create do that as well?